view/frontend/web/js/model/shipping-save-processor/myParcel.js

```diff
@@ -0,0 +1,17 @@
+define(
+    [
+        'jquery'
+    ],
```

Comment:
Could you combine this file and `view/frontend/web/js/mixins/checkout/model/shipping-save-processor/payload-extender-mixin.js` into `view/frontend/web/js/model/shipping-save-processor.js`? We are currently working with mixins and this would be in line with how we are implementing them, as well as make it more readable. 

Also, could you just use `document.querySelector('[name="delivery_options"]')` instead of jQuery?